lib/ui/views/explore/explore_section_view.dart

```diff
@@ -83,10 +84,11 @@ class ExploreSectionWidget extends StatelessWidget {
                   textAlign: TextAlign.center,
                 ),
                 const SizedBox(height: CustomPaddingSize.small),
-                DarkButton(
-                  'Explore',
-                  // TODO
-                  onPressed: () {},
+                FilledButton(
```

Comment:
@JElgar maybe we can still use custom buttons to add the colors and styles by default? For example here we can use `NowFilledButton` that I created

Reply:
We chatted on slack about this but the default styles are defined here https://github.com/now-u/now-u-app/blob/70ddb1dad9d0a7a90524250f5cd43c1273a70081/lib/themes.dart#L122 and those are applied to the built in flutter buttons